src/components/QueueMessage.js

```diff
@@ -50,22 +72,66 @@ class QueueMessage extends React.Component {
       return null
     }
 
+    const handleTabKeyPress = (e, tab) => {
```

Comment:
I think if we technically want to do accessibility right, we should follow the W3's example of how tabs should be implemented: https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html. See https://codepen.io/svinkle/pen/edmDF for another example. Basically, `Tab` should only tab into or out of the tab list, and when we're in the list, right/left arrow keys are how you navigate. Might be time to refresh my screen reader skills so I can test this well...

Reply:
Haven't done this one yet, seems to be a bit more involved than the other 2 changes so I'll see what I can do but go ahead and make any additions you want

Reply:
You can go ahead and hold off on this for now, we can do this later as part of a broader accessibility audit.